src/main/java/org/openstreetmap/atlas/geography/atlas/items/Area.java

```diff
@@ -77,6 +79,19 @@ public boolean intersects(final GeometricSurface surface)
         return surface.overlaps(asPolygon());
     }
 
+    @Override
+    public boolean isCountrySliced()
+    {
+        // Get the last 6 digits of the identifier
+        final String stringIdentifier = String.valueOf(this.getIdentifier());
```

Comment:
Do we need to check for empty String!!

Reply:
The identifier can't be empty, so there should always be a string.